dashboard/src/locales/messages/index.ts

```diff
@@ -112,6 +112,7 @@ export const messages = {
     'global.fails': 'Fails',
     'global.filters': 'Filters',
     'global.github': 'GitHub',
+    'global.hardware': 'Hardware',
```

Comment:
add the plural variant for the paginationInfo component
```suggestion
    'global.hardware': 'Hardware',
    'global.hardwares': 'Hardwares',
```